### ISSUE TITLE
iOS does not show proper scale svg image.

### ISSUE THREAD

**Issue by yeahg-dev** [unoplatform/uno.resizetizer#313]
## Current behavior

I'm struggling issue with svg file in iOS with Uno Single Project.
As far as I know, just referencing svg file path in `Image` choose the appropriate scaled png file that device needs(ex. @3x). but my app can not display the right scaled image, so Image quality is poor.

```xaml
 <Image Source="Assets/Icons/icon_plus.png" Width="50" Height="50" />
```

<img src="https://github.com/user-attachments/assets/d918192d-9f14-4c33-8c30-04b78f9010e2" width="300"/>

I followed the description in Uno Platform document
https://platform.uno/docs/articles/external/uno.resizetizer/doc/using-uno-resizetizer.html?tabs=singleproject%2CAndroid#using-svg-images-vs-png-images-with-svg-underneath
I added UnoImage build action in csroj.

```xml
// .csproj
<Project Sdk="Uno.Sdk">
 ...
  <ItemGroup>
    <UnoImage Include="Assets\Icons\*.svg" />
  </ItemGroup>
...
</Project>
```
But It didn't help.

Do I miss something when using svg file?

## Expected behavior

Svg Image should rendered without lower quality.

## How to reproduce it (as minimally and precisely as possible)

https://github.com/yeahg-dev/UnoSvgTest

## Environment

Nuget Package:
Uno Sdk : "5.2.175"

Package Version(s):

Affected platform(s):

- [x] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

**Comment by yeahg-dev** [comment]
Anyone else having the same problem?

**Comment by MartinZikmund** [comment]
@yeahg-dev Agreed, I have seen such behavior before myself. We will make sure to look into this
